### ISSUE TITLE
Build time exceeding 24 hours are not shown properly

### ISSUE THREAD

**Issue by ggardet** [openSUSE/open-build-service#7382]
# Issue Description
See libreoffice builds on aarch64: https://build.opensuse.org/packages/libreoffice/job_history/openSUSE:Factory:ARM/standard/aarch64
On March 07, 2019 09:55, the package took longer than 24 hours to build and it is displayed as 05 min.
For the build from April 05, 2019 15:41, we do not know if was 5 hours or 29 hours.

Expected Result
============
It should show 24 h 05 min in this case and should not reset when exceeding 24h.

How to Reproduce
===============
1. Build a package taking longer than 24h
2. Check build time info

Further Information
===============
*https://build.opensuse.org/packages/libreoffice/job_history/openSUSE:Factory:ARM/standard/aarch64

**Comment by ggardet** [comment]
Here is a great example: openSUSE:Factory:ARM/scala on armv6l.
`osc buildhist openSUSE:Factory:ARM scala standard armv6l`returns: 
```
time                  srcmd5                              vers-rel.bcnt    rev   duration
2019-03-29 03:53:40   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.1       1       5789
2019-04-21 12:39:28   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.2       1       6315
2019-05-03 20:43:21   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.3       1       4060
2019-06-13 18:26:59   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.4       1       3581
2019-07-20 16:53:42   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.5       1       6257
2019-11-29 07:44:01   a88518b57dbacf9350507f37d1f6bda1    2.10.6-1.6       1       9933
2019-12-07 15:47:32   d182da381796d12db5d98f15b12947e6    2.10.7-2.1       3     178657
2019-12-09 11:07:03   d182da381796d12db5d98f15b12947e6    2.10.7-2.2       3     117698
2019-12-14 09:47:34   d182da381796d12db5d98f15b12947e6    2.10.7-2.3       3     197029
2019-12-16 14:08:32   824a498868e4a5ed4209396611d7580c    2.10.7-3.1       4     121914
```
whereas https://build.opensuse.org/packages/scala/job_history/openSUSE:Factory:ARM/standard/armv6l shows much smaller values.

**Comment by ggardet** [comment]
The faulty part seems to be `humanize_time`: https://github.com/openSUSE/open-build-service/blob/master/src/api/app/helpers/webui/package_helper.rb#L65

**Comment by danidoni** [comment]
This is related to #8878 and was fixed on #9319
